CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [2.0.2]
```

Comment:
As the version will be set when we do the final release please change this to `[Unreleased]` for now.

```suggestion
## [Unreleased]
```